src/app/components/options-sidebar/time-clock/time-clock.component.spec.ts

```diff
@@ -19,7 +18,14 @@ describe('TimeClockComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should be created', () => {
+  it("should be created", () => {
     expect(component).toBeTruthy();
   });
+
+  it("should set showfileds as true", () => {
+    const show = true;
+    component.showFields = show;
+    component.getShowFields(show);
```

Comment:
It seems this function is changing showFields value. In this case, the field name should be:

setShowFields(show)

Reply:
Done 👌